File: 1469-minimum-number-of-steps-to-make-two-strings-anagram/minimum-number-of-steps-to-make-two-strings-anagram.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        

        tmap = dict()
        smap = dict()

        for c in  t:
            tmap[c] = tmap.get(c,0)+1
        ans = 0 
        for c in  s:
            smap[c] = smap.get(c,0)+1
            if smap[c]> tmap.get(c,0):
                ans +=1
            
        return ans

File: 1469-minimum-number-of-steps-to-make-two-strings-anagram/minimum-number-of-steps-to-make-two-strings-anagram.java
class Solution {
    public int minSteps(String s, String t) {
        Map<Character, Integer> tmap = new HashMap<>();
        Map<Character, Integer> smap = new HashMap<>();
        
        for (char c : t.toCharArray()) {
            tmap.put(c, tmap.getOrDefault(c, 0) + 1);
        }
        
        int ans = 0;
        
        for (char c : s.toCharArray()) {
            smap.put(c, smap.getOrDefault(c, 0) + 1);
            if (smap.get(c) > tmap.getOrDefault(c, 0)) {
                ans++;
            }
        }
        
        return ans;

        
    }
}